code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -761,6 +763,15 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 			message_admins("[ADMIN_LOOKUPFLW(usr)] turned [shielding_powered ? "on" : "off"] the [src] powered shielding.")
 			investigate_log("[key_name(usr)] turned [shielding_powered ? "on" : "off"] the [src] powered shielding.")
 			. = TRUE
+		if("reaction_suppression")
+			if(!nob_crystal_inserted)
+				message_admins("[ADMIN_LOOKUPFLW(usr)] tried to toggle reaction suppression on a canister without a noblium crystal inside, possible href exploit attempt.")
```

Comment:
stack_trace instead please, admins rarely have the knowledge to investigate these